packages/uikit/src/widgets/Farm/components/WithdrawModal/index.tsx

```diff
@@ -40,26 +44,33 @@ const WithdrawModal: React.FC<React.PropsWithChildren<WithdrawModalProps>> = ({
   const handleChange = useCallback(
     (e: React.FormEvent<HTMLInputElement>) => {
       if (e.currentTarget.validity.valid) {
-        setVal(e.currentTarget.value.replace(/,/g, "."));
+        const inputVal = e.currentTarget.value.replace(/,/g, ".");
+        setVal(inputVal);
+
+        const inputValUSDPrice = inputVal.trim() === "" ? "0.00" : new BigNumber(inputVal).times(lpPrice).toFixed(2);
+        setValUSDPrice(inputValUSDPrice);
       }
     },
-    [setVal]
+    [setVal, setValUSDPrice, lpPrice]
   );
 
   const handleSelectMax = useCallback(() => {
     setVal(fullBalance);
-  }, [fullBalance, setVal]);
+    setValUSDPrice(new BigNumber(fullBalance.replace(/,/g, "")).times(lpPrice).toFixed(2));
+  }, [fullBalance, setVal, setValUSDPrice, lpPrice]);
 
   const handlePercentInput = useCallback(
     (percent: number) => {
       const totalAmount = fullBalanceNumber
         .dividedBy(100)
         .multipliedBy(percent)
         .toNumber()
-        .toLocaleString("en-US", { maximumFractionDigits: decimals });
+        .toLocaleString("en-US", { maximumFractionDigits: decimals })
+        .replace(/,/g, "");
```

Comment:
`.toLocaleString("en-US", { maximumFractionDigits: decimals })`
`en-US` will not have `,` issue. So no need every place replace it.

Reply:
![image](https://user-images.githubusercontent.com/109973128/209660812-5942f996-2603-443b-986b-aa23a210bd70.png)
It's still have `,` , so need to replace👀